inc/miniRT.h

```diff
@@ -155,6 +159,9 @@ void		*parse_ambiant(char **values);
 
 t_camera	*camera(t_fvector3 position, t_fvector3 normal, int fov);
 void		*parse_camera(char **values);
+void		rotate_camera_yaw(t_camera *cam, float theta);
+void		rotate_camera_pitch(t_camera *cam, float theta);
+void		add_fov(t_minirt *mrt, int incrementation);
```

Comment:
```suggestion
void		update_yaw(t_camera *cam, float theta);
void		update_pitch(t_camera *cam, float theta);
void		update_fov(t_minirt *mrt, int incrementation);
```
je pense que c'est mieux non ?